server/apiRouter.js

```diff
@@ -16,6 +16,8 @@ const transactionLineItems = require('./api/transaction-line-items');
 const initiatePrivileged = require('./api/initiate-privileged');
 const transitionPrivileged = require('./api/transition-privileged');
 
+const createWithIdp = require('./api/auth/createWithIdp');
```

Comment:
Where's `authWithIdp`?

Reply:
Or is it omitted on purpose?

Reply:
We are using `authWithIdp` only internally in FTW server (e.g. here: https://github.com/sharetribe/ftw-daily/pull/1366/files#diff-c2569a98b269447cc716700472d37356R51) so there is no specific need for adding a route for it. 

First I had the authWithIdp under `api-util` instead of `api/auth` but that felt also confusing.